src/update.jl

```diff
@@ -360,3 +360,90 @@ function choosePartition(net::HybridNetwork)
 end
 
 
+"""
+updateUninformativeQuartets(obsCF::Array{Float64}, tol::Float64)
+Returns value to place in quartet.sampled
+"""
+function updateUninformativeQuartets(quartet::Quartet, atol::Float64)
+    allcomp = [(abs(a-b)<=atol) for a in quartet.obsCF', b in quartet.obsCF]
+    if false in allcomp
+        return(true)
+    else
+        return(false)
+    end
+end
+
+"""
+updateUninformativeQuartets!(quartets::Vector{Quartet}, tol::Float64)
+Checks for quartets classified as 'uninformative' given a tolerance to define CF equality 
+    if all CFs in quartet.obsCF are equal within 'tol' tolerance, 
+    quartet.sampled is set to 'false'
+Output: Updates quartet.sampled in-place for all quartets 
+"""
+function updateUninformativeQuartets!(quartets::Vector{Quartet}, atol::Float64)
+    i = Threads.Atomic{Int}(0);
+    Threads.@threads for q in quartets
+        q.sampled = updateUninformativeQuartets(q, atol)
+        if !(q.sampled)
+            #println("bad")
+            q.uninformative = true
+            Threads.atomic_add!(i, 1)
+        end
+    end
+    return(i[])
+end
+updateUninformativeQuartets!(d::DataCF, atol::Float64) = updateUninformativeQuartets!(d.quartet, atol)
+
+"""
+updateUninformativeQuartetsAll!(quartets::Vector{Quartet}, toset::Bool)
+Updates in-place all Quartet.uninformative values to the value provided in toset::Bool
+"""
+function updateUninformativeQuartetsAll!(quartets::Vector{Quartet}, toset::Bool)
+    Threads.@threads for q in quartets
+        q.uninformative=toset
+    end
+end
+updateUninformativeQuartetsAll!(quartets::Vector{Quartet}) = updateUninformativeQuartetsAll!(quartets, false)
+updateUninformativeQuartetsAll!(d::DataCF, toset::Bool) = updateUninformativeQuartetsAll!(d.quartet, toset)
+updateUninformativeQuartetsAll!(d::DataCF) = updateUninformativeQuartetsAll!(d.quartet, false)
+
+"""
+updateSampledQuartetsAll!(quartets::Vector{Quartet}, toset::Bool)
+Updates in-place all Quartet.sampled values where Quartet.uninformative=false 
+    to the value provided in toset::Bool
+"""
+function updateSampledQuartetsAll!(quartets::Vector{Quartet}, toset::Bool)
+    Threads.@threads for q in quartets
+        if !q.uninformative
+            q.sampled=toset
+        end
+    end
+end
+updateSampledQuartetsAll!(quartets::Vector{Quartet}) = updateSampledQuartetsAll!(quartets, true)
+updateSampledQuartetsAll!(d::DataCF, toset::Bool) = updateSampledQuartetsAll!(d.quartet, true)
+updateSampledQuartetsAll!(d::DataCF) = updateSampledQuartetsAll!(d.quartet, true)
+
+"""
+updateSubsetQuartets!
```

Comment:
`randQuartets` currently does not have any coverage but it seems to do similar things and I wonder if this function could effectively replace it and kill two birds with one stone.